src/util.rs

```diff
@@ -1,15 +1,14 @@
-use crate::error::OutOfBounds;
 use sha2::{Digest, Sha256};
 use std::any::type_name;
 
-pub fn get_struct_name<T>() -> Result<String, OutOfBounds> {
-    Ok(type_name::<T>()
+pub fn get_struct_name<T>() -> String {
+    type_name::<T>()
         .split("::")
         .collect::<Vec<&str>>()
         .get(2)
```

Comment:
Actually why are we getting position 2?

Reply:
Okay just checked the code, since I didn't really looked at it before after it got merge into the initial pull request.

So, for example, given a pod, type_name::<Pod>() returns "orcapod::model::Pod", so we would need to extract the last section and ignore the rest. Given that, this should have been implemented with last and not get(2). I will make a change quickly